packages/shared/lib/services/sync/run.service.integration.test.ts

```diff
@@ -209,12 +218,18 @@ describe('SyncRun', () => {
     });
 });
 
+const initDb = async () => {
+    await database.multipleMigrations();
+    await migrateRecords();
+};
+
 const clearDb = async () => {
     await db.knex.raw(`DROP SCHEMA nango CASCADE`);
+    // TODO: clear records???
```

Comment:
I don't really want to expose records.db outside of the package, especially if it is only used in tests. any idea?

Reply:
done in [75f759d](https://github.com/NangoHQ/nango/pull/2030/commits/75f759dd86aa28095d0f83fa876c4962a9f7dc47)